lib/Echidna/Types/Cache.hs

```diff
@@ -0,0 +1,7 @@
+module Echidna.Types.Cache where
+
+import Data.Map (Map)
+import EVM.Types (W256, Addr, Contract)
+
+type ContractCache = Map Addr (Maybe Contract)
+type SlotCache = Map Addr (Map W256 (Maybe W256))
```

Comment:
Can you use these new types in Types/Config.hs:Env definition?

Reply:
Done